build/yaml/crd/vpc/crd.nsx.vmware.com_securitypolicies.yaml

```diff
@@ -448,7 +448,6 @@ spec:
                             description: Port is the name or port number.
                             x-kubernetes-int-or-string: true
                           protocol:
-                            default: TCP
```

Comment:
why we have this CRD change?

Reply:
They are generated after I upgrade controller-gen to v.0.16.5. Is it an expected behavior? Or we shouldn't let controller-gen to modify the crd?